WcaOnRails/app/webpacker/components/Panel/Wfc/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  panelWfcPageDataUrl,
+  countryBandsUrl,
+} from '../../../lib/requests/routes.js.erb';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import Errored from '../../Requests/Errored';
+import Loading from '../../Requests/Loading';
+import DelegateProbations from '../../DelegateProbations';
+import DuesExport from './DuesExport';
+import PanelTemplate from '../PanelTemplate';
+
+const sections = [
+  {
+    id: 'dues-export',
+    name: 'Dues Export',
+    component: DuesExport,
+  },
+  {
+    id: 'country-bands',
+    name: 'Country Bands',
+    link: countryBandsUrl,
+  },
+  {
+    id: 'delegate-probations',
+    name: 'Delegate Probations',
+    component: DelegateProbations,
+    forAtleastSeniorMember: true,
+  },
+];
+
+export default function Wfc() {
+  const { data, loading, error } = useLoadedData(panelWfcPageDataUrl);
+
+  if (loading) return <Loading />;
+  if (error) return <Errored />;
```

Comment:
Would it be worth assuming that they aren't a senior member if this errors, rather than showing the error component? If I'm a senior and this errors, then I can't do anything, but if it assumed I wasn't instead then I could still access country bands and other public information. (With some error message somewhere to indicate why they can't see the other stuff.)

Reply:
I'm not sure whether you understood it correctly, so the use-case here is that the WFC panel will be accessible to all WFC members irrespective of whether they are senior members or not. Only difference is, if they are senior member, they will get an additional option 'modify delegate probation'.

Reply:
I think I understand. I'm suggesting that if there's an error, then show `PanelTemplate` and assume `data.isAtleastSeniorMember` is false (plus show some error message _somewhere_), rather than omitting the component and only showing `Errored`. This way, if there's an error, the user can still access the standard options that all WFC members see.

Reply:
Oh okay, got it. Thanks for clarifying, sorry for not getting it initially. This looks like a good idea, I implemented this.